source/process/Recruitment

```diff
@@ -0,0 +1,137 @@
+Our mission at Mattermost is to be the primary way people communicate within Enterprise and applications. 
+ Our goals are to become the leading enterprise-grade message solution, we’ll achieve this by: 
+#1 private cloud solution by deployments (12-18 months)
+#1 private cloud solution by revenue
+#1 solution by deployments (private cloud & SaaS)
+#1 solution by revenue (private cloud & SaaS)
+
+Our Values
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values:
+
+Customer Obsession - We exist to make customers successful. In everything we do, start with customer perspective and work backwards. Earn and keep their trust.
+Ownership - Own the outcomes of your activity. Don’t drop the ball. When we see a vacuum on something important, we jump in–we never say “it’s not my job.”
+High Impact - Figure out what matters most, and focus on those priorities. When deciding to work on low impact or high impact projects, choose high impact.
+Self-Awareness - We understand and seek to understand our strengths and growth opportunities, as individuals and organizations. We are open to critique and share critique constructively and respectfully.
+Be proud of what you build - Insist on high standards. Ensure that you’re building the highest quality products, services, and processes possible, and constantly raising the bar.
+
+
+
+At Mattermost we hire the best of the best to help us attain our mission while portraying our values.  Our recruitment process follows 8-9 steps depending on the role.  
+ 
+Non-Engineering Recruitment Process: 
+
+1. Applicant Review 
+
+Recruiter(s) will review inbound and network referral applications for all assigned requisitions and will archive/screen as is appropriate
```

Comment:
@lindy65 should we edit this to be a consistent tense/voice? 

ie
"Recruiter(s) review inbound..."
"Hiring manager reviews shortlist..."